src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -22,4 +22,13 @@ public User get(String login) {
         }
         return null;
     }
+
```

Comment:
The `get` method should return an `Optional<User>` instead of `null`. This change will help prevent potential `NullPointerException` and align with best practices for handling absent values.